Library/Homebrew/os/mac/xquartz.rb

```diff
@@ -98,6 +99,9 @@ def prefix
           Pathname.new("/opt/X11")
         elsif Pathname.new("/usr/X11/lib/libpng.dylib").exist?
           Pathname.new("/usr/X11")
+        # X11 doesn't include libpng on Tiger
+        elsif File.exist?("/usr/X11R6/lib/libX11.dylib")
```

Comment:
Style nit. Probably want to use same `Pathname#exist?` or vice verse to keep the consistence.